salt/journal/config/etc-nginx-traits.d-redirect-existing-paths.conf

```diff
@@ -1435,7 +1435,8 @@ map $uri $new_uri {
     '/category/microbiology-and-infectious-disease' '/subjects/microbiology-infectious-disease';
     '/category/neuroscience' '/subjects/neuroscience';
     '/category/plant-biology' '/subjects/plant-biology';
-    '/privacy-policy' '/terms';
+    '/privacy-notice' '/privacy';
```

Comment:
EJP is going to use `/privacy-notice`, but it's simpler to add a redirect than get it updated.